src/lib/journey/stages/stages.stories.js

```diff
@@ -80,80 +80,80 @@ RegistrationInteraction.play = async ({ canvasElement }) => {
   const news = canvas.getByRole('checkbox', { name: /news/ })
   const securityQuestion = canvas.getByLabelText('Select a security question')
   const securityAnswer = canvas.getByLabelText('Security Answer')
-  const toc = canvas.getByRole('checkbox', { name: 'Please accept our Terms and Conditions' });
+  const tocLink = canvas.getByRole('link', { name: 'View full Terms & Conditions' });
+  const toc = canvas.getByRole('checkbox', { name: 'Please accept our Terms & Conditions' });
 
   const usernameCb = frRegistrationStep.getCallbacksOfType(CallbackType.ValidatedCreateUsernameCallback)[0];
   const passwordCb = frRegistrationStep.getCallbacksOfType(CallbackType.ValidatedCreatePasswordCallback)[0];
   const firstNameCb = frRegistrationStep.getCallbacksOfType(CallbackType.StringAttributeInputCallback)[0];
-  console.log(frRegistrationStep)
   const lastNameCb = frRegistrationStep.getCallbacksOfType(CallbackType.StringAttributeInputCallback)[1];
   const emailCb = frRegistrationStep.getCallbacksOfType(CallbackType.StringAttributeInputCallback)[2];
   const specialOffersCb = frRegistrationStep.getCallbacksOfType(CallbackType.BooleanAttributeInputCallback)[0]
   const newsOffersCb = frRegistrationStep.getCallbacksOfType(CallbackType.BooleanAttributeInputCallback)[1]
-
   const securityQuestions = frRegistrationStep.getCallbacksOfType(CallbackType.KbaCreateCallback)[0];
-
   const tocCb = frRegistrationStep.getCallbacksOfType(CallbackType.TermsAndConditionsCallback)[0]
 
-  expect(username).toHaveFocus();
+  await expect(username).toHaveFocus();
```

Comment:
My VS Code was barking about this not having an `await`, likely from my Storybook extension. We should add awaits to the rest of the tests as well over time.

Reply:
yeah ive been ignoring these errors because i don't understand why a non promise i needs to be awaited?

Reply:
Okay, after some digging, I found the reason: https://github.com/storybookjs/eslint-plugin-storybook/blob/main/docs/rules/await-interactions.md.

It looks like the original implementations of the Testing Library methods are wrapped by Storybook and to provide the full functionality of Storybook's Interactions, they recommend using `await`.